### PR TITLE
Add purism's mobile version of the evince package

### PR DESCRIPTION
**Pull request by prurigro**

### CODE REVIEW

**Comment by prurigro** [comment]
The desktop version in the repos doesn't fit on the pinephone's screen nicely like this one does

File: PKGBUILDS/phosh/evince-mobile/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+pkgver=3.34.1
+pkgrel=0
+pkgdesc='Purism mobile document viewer (PDF, Postscript, djvu, tiff, dvi, XPS, SyncTex support with gedit, comics books (cbr,cbz,cb7 and cbt))'
+url='https://source.puri.sm/Librem5/evince'
+arch=('aarch64')
+license=('GPL')
+_commit=2fa24ebae9deb0fabb5dd50dff523ab7a79d2237
+source=("git+https://source.puri.sm/Librem5/evince.git#commit=$_commit")
+sha256sums=('SKIP')
+provides=(libev{document,view}3.so)
```
Comment (Danct12) [inline PKGBUILDS/phosh/evince-mobile/PKGBUILD:16]: we probably should also provide the original evince package, so we don't get conflicts.

Comment (prurigro) [reply PKGBUILDS/phosh/evince-mobile/PKGBUILD:16]: good call!

**Comment by Danct12** [comment]
Merged in Huong Tram Linux branch.

**Pull request closed without merging**